lua/autorun/server/sv_chip_lister.lua

```diff
@@ -68,6 +68,79 @@ util.AddNetworkString( "CFC_ChipLister_UpdateListData" )
 util.AddNetworkString( "CFC_ChipLister_ToggleHUD" )
 
 
+-- Get the four corners of a thin, flat plate.
+local function getPlateCorners( ent )
```

Comment:
This can be cached per run, this currently runs multiple times per tick, can be reduced to one per tick